scholly/service/core/build.js

```diff
@@ -38,6 +38,7 @@ async function buildBaseApp(data) {
         [REACT_APP_TEAM_PUBLICATIONS]: JSON.stringify(data.teamPublications),
         [REACT_APP_TEAM_INFO]: JSON.stringify(data.teamInfo),
         [REACT_APP_TEAM_MEMBERS]: JSON.stringify(data.teamMembers),
+        [REACT_APP_DEPLOYING]: 'true',
```

Comment:
Do we need this? Why expose the env var to the scholly service if it's just used for debugging. We could just set the environment variable manually and run the base app w/out going through Scholly, if its for debugging purposes.

Reply:
For linux/mac you could export:
`export REACT_APP_DEBUG=true && npm start`

Reply:
So to clarify:
IMO we should just let the base react app know about the debug environment variable so we won't have to make changes to the scholly service. This can be achieved by setting the env var manually before running the base app.